packages/assets-controllers/src/CurrencyRateController.test.ts

```diff
@@ -269,14 +272,51 @@ describe('CurrencyRateController', () => {
       messenger,
       state: { currentCurrency: 'xyz' },
     });
-
+    controller.start();
     await controller.updateExchangeRate();
 
     expect(controller.state.conversionRate).toStrictEqual(2000.42);
 
     controller.destroy();
   });
 
+  it('should fetch exchange rates after calling setNativeCurrency by default', async () => {
+    const fetchExchangeRateStub = sinon.stub().resolves({});
+    const messenger = getRestrictedMessenger();
+    const controller = new CurrencyRateController({
+      includeUsdRate: true,
+      fetchExchangeRate: fetchExchangeRateStub,
+      messenger,
+    });
+
+    controller.start();
+    await controller.setNativeCurrency('XYZ');
+
+    expect(
+      fetchExchangeRateStub.alwaysCalledWithExactly('usd', 'XYZ', true),
+    ).toBe(true);
+
+    controller.destroy();
+  });
+
```

Comment:
Maybe add a test case for the new behavior of requiring `start` to actually start?

```suggestion
  it('should NOT fetch exchange rates after calling setNativeCurrency if start has not been called', async () => {
    const fetchExchangeRateStub = sinon.stub().resolves({});
    const messenger = getRestrictedMessenger();
    const controller = new CurrencyRateController({
      includeUsdRate: true,
      fetchExchangeRate: fetchExchangeRateStub,
      messenger,
    });
    fetchExchangeRateStub.resetHistory();
    await controller.setNativeCurrency('XYZ');

    expect(fetchExchangeRateStub.notCalled).toBe(true);

    controller.destroy();
  });
```